queries/useApyHistoryQuery.ts

```diff
@@ -1,43 +1,10 @@
 import { useQuery } from "react-query";
-import { QUERY_KEYS, apyDayOptions } from "../constants";
-import moment from "moment";
+import { QUERY_KEYS } from "../constants";
 
-async function fetchApyHistory() {
-  const apyHistory = await Promise.all(
-    apyDayOptions.map(async (days) => {
-      const endpoint = `${process.env.NEXT_PUBLIC_ANALYTICS_ENDPOINT}/api/v2/oeth/apr/trailing_history/${days}`;
-      const response = await fetch(endpoint);
-      if (!response.ok) {
-        throw new Error(`Failed to fetch ${days}-day trailing APY history`);
-      }
-      const json = await response.json();
-      return json.trailing_history;
-    })
-  ).catch(function (err) {
-    console.log(err.message);
-  });
-
-  const data = {};
-
-  apyDayOptions.map((days, i) => {
-    data[`apy${days}`] = apyHistory ? apyHistory[i] : [];
-  });
-
-  Object.keys(data).map((key) => {
-    data[key] = data[key]
-      .filter((item) => moment(item.day).isAfter(moment().subtract(14, "days")))
-      .reverse();
-  });
-
-  // Remove last apy day when apy days for trailing average
-  data?.["apy7"]?.pop();
-  data?.["apy30"]?.pop();
-
-  return data;
-}
+import { fetchApyHistory } from "../utils";
 
 const useApyHistoryQuery = (apyHistory, options = {}) => {
-  return useQuery(QUERY_KEYS.ApyHistory(365), fetchApyHistory, {
+  return useQuery(QUERY_KEYS.ApyHistory(365), () => fetchApyHistory(), {
```

Comment:
What effect does this change have?

Reply:
it was a typescript thing. I guess useQuery expects the fetch function to follow a certain convention